sdk/src/types/block/output/account.rs

```diff
@@ -617,7 +617,7 @@ pub(crate) mod dto {
     /// Describes an account in the ledger that can be controlled by the state and governance controllers.
     #[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
     #[serde(rename_all = "camelCase")]
-    pub struct AccountOutputDto {
+    pub(crate) struct AccountOutputDto {
```

Comment:
This could be private. I know it's for the test but maybe it could just do a JSON back and forth and it would achieve the same. Maybe later.